src/CHIPBindings.cc

```diff
@@ -1863,7 +1863,7 @@ hipError_t hipRuntimeGetVersion(int *RuntimeVersion) {
   NULLCHECK(RuntimeVersion);
 
   if (RuntimeVersion) {
-    *RuntimeVersion = 1;
+    *RuntimeVersion = CHIPSTAR_MAJOR_VERSION*10000000+CHIPSTAR_MINOR_VERSION*100000+CHIPSTAR_PATCH_VERSION;
```

Comment:
A style nit from clang-format:
```suggestion
    *RuntimeVersion = CHIPSTAR_MAJOR_VERSION * 10000000 +
                      CHIPSTAR_MINOR_VERSION * 100000 + CHIPSTAR_PATCH_VERSION;
```